EventEnabledInsurance/post-load-test-readme.md

```diff
@@ -0,0 +1,99 @@
+# Overview
+This load test script will add 1000 messages to the MQ Queue `Quote` by making REST calls via IBM API Connect. These messages will start appearing in the MQ queue named `Quotes` and the IntegrationServer `ace-db-writer-int-srv-eei` will read from the queue, one message per second and add them to the [System Of Record database](readme.md#working-directly-with-the-system-of-record-database).
```

Comment:
Docs look good. I was expecting this to be added to the end of the readme, but admittedly the readme is getting out of control. Maybe include a link to this from the readme.md? Maybe later we can decide if we want to split the other stories out of the readme as well.

Reply:
Done